### ISSUE TITLE
Feature request: configure which devices are used for different sound categories

### ISSUE THREAD

**Issue by JuliusSweetland** [OptiKey/OptiKey#703]
It would be useful to be able to select which devices are used for categories of sounds, such as system sounds (selection beeps etc) vs voices. This would allow a user to listen to some functional system sounds, but output voices via a speaker, for example.

**Comment by JuliusSweetland** [comment]
More details: "I mean being able to listen to YouTube videos, online series, all the multimedia on the PC. All that in bluetooth. Then just hear everything about Optikey out loud"

**Comment by kmcnaught** [comment]
This would be good. At the same time, it sounds like there are currently some foibles in the sound outputs, as per: https://github.com/OptiKey/OptiKey/issues/569

**Comment by bigmutty** [comment]
I agree. I sometimes like to use bluetooth earphones to listen to music and any other system notifications. On Tobii Communicator This feature exists, but every time I connect my earphones I have to go into the settings and reapply the playback audio device because it will switch to the earphones. I assume they reference audio devices by the Id and not the name.

**Comment by bigmutty** [comment]
I know there was a way to select which device to use in SAPI 5, but that was years ago. Could you point me to the code initiating the Microsoft voice engine?
